src/addTaskUI.js

```diff
@@ -0,0 +1,64 @@
+import moveSrc from './move.png';
+import trashSrc from './trash.png';
+import clearTask from './remove.js';
+
+const toDoListUI = document.querySelector('.to-do-list');
+const endCont = document.querySelector('.clear');
+
+const appendTask = (item, list) => {
+  const taskUI = document.createElement('li');
+  taskUI.innerHTML = `<div><input type='checkbox' id = task-${item.index} class = 'check' value=${item.index}><p size='12'>${item.description}</p></div>`;
+  const moveImg = new Image();
+  moveImg.src = moveSrc;
+  taskUI.appendChild(moveImg);
+  moveImg.addEventListener('click', () => {
+    if (moveImg.src === moveSrc) {
+      moveImg.src = trashSrc;
+      moveImg.parentNode.style.background = 'rgba(214, 214, 148, 0.534)';
+      taskUI.children[0].children[1].setAttribute('contenteditable', 'true');
+      endCont.style.fontSize = '25px';
+      endCont.style.color = 'rgba(243, 9, 9, 0.534)';
+      endCont.addEventListener('click', (e) => {
+        if (moveImg.src === trashSrc) {
+          e.target.textContent = 'Clear all completed';
+          e.target.style.background = 'rgba(168, 157, 157, 0.534)';
+          e.target.style.fontSize = '16px';
+          e.target.style.color = 'black';
+          taskUI.children[0].children[1].setAttribute('contenteditable', 'false');
```

Comment:
- [ ] Good job implementing the required functions, but at the moment when you edit the task it does not persist in the localStorage. Please kindly ensure that it persists in the localStorage when it is edited.
- [ ] [OPTIONAL] Please also work on making sure the browser does not reload every time a task is added as it does not create a good user experience.